stack-ghc822.yaml

```
@@ -1,6 +1,3 @@
-flags: {}
+resolver: lts-11.22
```

```suggestion
resolver: lts-11.0
```
The point here is that we would like to test all changes for their backwards-compatibility. As such the most useful LTS version to test against is the initial release. The idea here is that if something builds against `lts-11.0`, it is guaranteed (?) to work with `lts-11.X` for any valid `X`. On the other hand, it is possible to have something successfully build against `lts-11.22` but not against `lts-11.0` (e.g. if there is a library `lib` in both `lts-11.0` and in `lts-11.22`, but the version of `lib` included in `lts-11.22` has additional functions - then if we use those additional functions, we'll be able to build successfully against `lts-11.22` but fail against `lts-11.0`).

---

Well, as you wish, but I do not find this reasoning particularly convincing. It is extremely rare for a package to e.g. work with `lts-11.X` and stop working for `lts-11.(X+1)`. Most likely you are just choosing more buggy snapshot of the ecosystem.

---

>  It is extremely rare for a package to e.g. work with `lts-11.X` and stop working for `lts-11.(X+1)`.

That's exactly my point - this is why I think it makes sense to build against `lts-11.0`. At least if packages adhere to the PVP, anything that builds against `lts-11.0` will, by induction, build against `lts-11.X` for any `X`.